Sources/SwiftUIBackports/iOS/PresentationDetents/InteractiveDetent.swift

```diff
@@ -4,8 +4,13 @@ import SwiftUI
 @available(iOS, deprecated: 16.0)
 public extension Backport where Wrapped: View {
 
-    /// The identifier of the largest detent that is not dimmed.
+    /// Removes dimming from detents higher than the provided identifier.
```

Comment:
I had trouble with this so I appreciate this. However shouldn't this be:

`Removes dimming from detents higher (and including) the provided identifier`

Thoughts?

Reply:
Yes you are correct! Sorry about that, I described it correctly at the parameter description but not here.

Could you update your comment to be a `suggestion` so that I can accept it directly in GitHub.

Reply:
Good point, I forgot 👍